modules/mobile/app/serializers/mobile/v0/debts_serializer.rb

```diff
@@ -36,34 +37,28 @@ def dependent_debts?(debts)
       end
 
       def serialize_debt(debt, id = nil)
-        DebtStruct.new(id: id || debt['id'],
-                       fileNumber: debt['fileNumber'],
-                       payeeNumber: debt['payeeNumber'],
-                       personEntitled: debt['personEntitled'],
-                       deductionCode: debt['deductionCode'],
-                       benefitType: debt['benefitType'],
-                       diaryCode: debt['diaryCode'],
-                       diaryCodeDescription: debt['diaryCodeDescription'],
-                       amountOverpaid: debt['amountOverpaid'],
-                       amountWithheld: debt['amountWithheld'],
-                       originalAR: debt['originalAR'],
-                       currentAR: debt['currentAR'],
-                       debtHistory: debt['debtHistory'])
+        debt_history = Array.wrap(debt['debtHistory']).map do |history|
+          {
+            date: history['date'],
+            letter_code: history['letterCode'],
+            description: history['description']
+          }
+        end
+
+        Debt.new(id: id || debt['id'],
+                 file_number: debt['fileNumber'],
```

Comment:
Is it ok if any of these debt attributes are nil? Do you need a default value?

Reply:
Yes nil is ok for the values